src/main/scala/wdl4s/types/WdlPairType.scala

```diff
@@ -1,11 +1,15 @@
 package wdl4s.types
 
+import spray.json.JsObject
 import wdl4s.values.{WdlPair, WdlValue}
 
+import scala.language.postfixOps
+
 case class WdlPairType(leftType: WdlType, rightType: WdlType) extends WdlType {
 
   override def isCoerceableFrom(otherType: WdlType): Boolean = otherType match {
     case WdlPairType(otherType1, otherType2) => leftType.isCoerceableFrom(otherType1) && rightType.isCoerceableFrom(otherType2)
+    case jsObject: JsObject if jsObject.fields.size.equals(1) => rightType.coerceRawValue(jsObject.fields.head._2).isSuccess
```

Comment:
This logic seems kinda weird?
Also, `otherType` is a `WdlType` so will never be `JsObject`. Presumably you don't need a case for this here.